config/xsl/trendreport/sections/navigation.xsl

```diff
@@ -5,7 +5,7 @@
         <nav>
```

Comment:
Do we have to add the responsive nav stuff here as well?

Reply:
Will do.